styles.css

```diff
@@ -87,3 +88,249 @@ body {
   padding-left: 10px;
   padding-right: 10px;
 }
+
+.Middle-page {
+  background-color: #e5e5e5;
+  height: 100%;
+  margin-top: -35px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.Middle-page > h1 {
+  text-align: center;
+  padding-top: 100px;
+  padding-bottom: -10px;
+  font-size: 40px;
+  font-family: 'Crete Round', serif;
+}
+
+.stand-alone {
+  padding-top: 50px;
+  font-family: 'Crete Round', serif;
```

Comment:
- [ ] It would be better if we implement the CSS DRY principle; this principle helps keeps our codebase clean and easy to maintain. For example; `display:flex; align-items: center; font-family: 'Crete Round', serif; font-family: 'Inter', sans-serif;` and a few of the style rules have been repeated. Let us kindly fix them. We can achieve this in this way, for example;
```
.class1, .class2, .class2 { display: flex}
```